ita-social-projects/StreetCode_Client
# Feature/issue 977/news pagination

### How to review
- go to `admin-panel/news`
- open dev tools and go to `Network` tab
- switch pages and verify that requests to api is sent to endpoint `news/getAll`;
- do some modification and check if news in table are updated;
